app/site.rb

```diff
@@ -217,21 +223,31 @@ def makefile?
 	protected
 
 		def maybe_commit(git, message, user)
-			git.status.each do |f|
-				if f.type
-					if user && user.email
-						git.commit("#{message} via tint", author: "#{user.fn} <#{user.email}>")
-					else
-						git.commit("#{message} via tint")
-					end
+			begin
+				status = git.status
+			rescue Git::GitExecuteError
```

Comment:
Yes, if there are no commits status will fail, so we know we can't be doing an empty commit attempt (since we added something that cannot have already been there -- nothing was there) so we just make the commit.